.github/workflows/test.yml

```diff
@@ -15,12 +22,30 @@ jobs:
           'Django~=3.0.0',
           'Django~=3.1.0',
         ]
+        services: ['']
+        additional-dependencies: ['']
         continue-on-error: [false]
 
         include:
           - python-version: 3.8
             django-version: 'https://github.com/django/django/archive/master.zip'
             continue-on-error: true
+          - python-version: 3.8
+            django-version: 'Django~=3.0.0'
+            services: postgresql-db
+            additional-dependencies: psycopg2
+            env:
+              DJANGO_DATABASE_ENGINE: django.db.backends.postgresql
+              DJANGO_DATABASE_PORT: 5432
+            continue-on-error: false
+          - python-version: 3.8
+            django-version: 'Django~=3.0.0'
+            services: mysql-db
```

Comment:
Do we need both `services` and `docker-compose.yml`? Can you please explain how this works?

Reply:
`services` is just a name of the matrix's variable, I named it so because it's related to `docker-compose` services needed to run tests. It's totally not related to GitHub action's services.
I can rename it if it's confusing, do you have any idea for a new name?

Reply:
`docker_compose_service`?